### ISSUE TITLE
Idea: create a iOS Shortcut automation 

### ISSUE THREAD

**Issue by nicolasglg** [blopker/superduper#20]
Hello,

this is a great app! Thank you 🙏
An idea, I created a iOS Shortcut automation that when my iPhone detect a nfc tag on the battery it opens Superduper app for 5 seconds (to unlock mode 4) then lock the iPhone. However to execute the iPhone needs to be unlocked in order to open the app. 

It would be great to have an integration of iOS shortcut so iPhone could stay locked and the action is set in background. 

This way we could trigger the shortcut while having the phone in the pocket and have almost a background mode like android.  

Best
Nico.

**Comment by JacopoV** [comment]
That reminds me how a widget for Android (to achieve a similar result) would be amazing to have :)

**Comment by blopker** [comment]
Thanks for the suggestion! I've never made an iOS shortcut before, but with the new physical iPhone shortcut button, that could be interesting. Although, I'm not clear on how an Android widget will make the experience better than just tapping the icon.

**Comment by blopker** [comment]
Apparently, you can have iOS open an app whenever it connects to a particular Bluetooth device... https://www.reddit.com/r/shortcuts/comments/104nek2/using_a_bluetooth_trigger_in_an_automation/ I haven't been able to get it actually working though.

**Comment by nicolasglg** [comment]
Yeah, I have tried to use the bluetooth connection as a trigger but the Super73 bluetooth is not eligible as it is not properly paired to the iPhone.
This is why I have put a NFC tag (10cts on amazon) that I sticked to the bike. When I approach my phone it automatically open Superduper, unfortunately I need to look at the phone so it unlocks, then locks it again.

It would be awesome if you could create shortcuts for ACTIVATE:
- Light ON
- Assist 4
- Mode 4

So this way we could to the following trigger if XXX is detected (Tag or wifi or location whatever) then Activate Light ON + Activate Assist 4 & so on. This would all be executed in background so the phone would stay in the pocket and it would be done seamlessly :)

**Comment by nicolasglg** [comment]
If you are able to generate iOS shortcut activation actions, then I can make a iOS shortcut automation that you can propose in the app later if you want.

**Comment by nicolasglg** [comment]
Here is some examples of app actions (sorry it is in French) : ![Capture d’écran 2023-09-30 à 11 39 35](https://github.com/blopker/superduper/assets/142723292/508bad73-6193-4c2d-ba30-f18300b0e4a9)

Here is my automation with Duperduper : 
![Capture d’écran 2023-09-30 à 11 39 14](https://github.com/blopker/superduper/assets/142723292/ee517f48-186c-4557-a055-80172e120c4d)

**Comment by Alexntf** [comment]
Hello ! Little question, What are the different mode exactly ? What the kmh limit for each one ? 
Thank you team!!

**Comment by blopker** [comment]
It's just the normal modes. They are coped in the app here: https://github.com/blopker/superduper/blob/main/lib/help.dart

You should join the super73 discord for more general help though!

**Comment by Alexntf** [comment]
Hey ! Thank you for your help !
It s my first s73 and first day ahah

I do not find the discord link on internet. 

And thank you for your work 🏍️
